bin/dev

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
```

Comment:
Since macOS uses port 5000, how about adding this line to make the server work on port 3000 instead?

```suggestion

export PORT=3000
```

Reply:
@pablobm Should we add this to the `.sample.env` as well?

Reply:
Uh, good shout. Perhaps that's the correct place to put it, instead of here. I just checked and Foreman does pick it up if it's in `.env`.

The downside would be that people are going to forget to provide a `.env` (or run `bin/setup` which does it for you) and will get this problem. I still think `.env` is the correct way to go as it allows people to select their desired port if they prefer it to be different, but we may need a troubleshooting section in the docs to warn about this.